configs/boost_timer.cfg

```diff
@@ -0,0 +1,14 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+dependencies = {
+    "required": ["boost"],
+}
+
+config = lsst.sconsUtils.ExternalConfiguration(
+    __file__,
+    headers=["boost/timer/timer.hpp"],
+    libs={"main": [], "test": ["boost_timer"]},
```

Comment:
I *think* makes it so that the library is only linked during unit tests. Is that the only time it's being used? If so, it's fine to start with this. If it's needed during normal execution time you can just use `libs=["boost_timer"]` instead of the dict.

Reply:
> I think makes it so that the library is only linked during unit tests.

Yes, that's what it does.

Reply:
Yes, that's the only time it's used, and the only time it's likely to be used. Per the [package description](https://www.boost.org/doc/libs/1_75_0/libs/timer/doc/):
> "How long does my C++ code take to run?" The Boost Timer library answers that question and does so portably.